File: 1848-sum-of-unique-elements/1848-sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        dict = {}
        for ele in nums:
            dict[ele] = 1 if ele not in dict else dict.get(ele,0)+1
        return sum([x for x in dict.keys() if dict[x] == 1])

File: 1848-sum-of-unique-elements/1848-sum-of-unique-elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
       HashMap<Integer, Integer>dict = new HashMap<Integer, Integer>();
		for(int num : nums){
		    if (dict.containsKey(num)==true)
		        dict.put(num, dict.get(num)+1);
		    else
		        dict.put(num, 1);
		}
		
		ArrayList<Integer> unique = new ArrayList<Integer>();
		for(int i : dict.keySet()){
		    if(dict.get(i) == 1)
		        unique.add(i);
		}
		
		int output = 0;
		for(int i=0; i<unique.size();i++)
		    output += unique.get(i);
		 
		return output; 
    }
}